tests/rkt_ace_validator_test.go

```diff
@@ -51,20 +49,14 @@ func TestAceValidator(t *testing.T) {
 	rktCmd := fmt.Sprintf("%s %s", ctx.cmd(), rktArgs)
 
 	t.Logf("Command: %v", rktCmd)
-	child, err := gexpect.Spawn(rktCmd)
-	if err != nil {
-		t.Fatalf("Cannot exec rkt: %v", err)
-	}
+	child := spawnOrFail(t, rktCmd)
```

Comment:
This will print the command twice, once in the line above and once inside spawnOrFail.